main/modes/mode_tiltrads.c

```diff
@@ -1186,9 +1186,11 @@ const song_t gameMusic = {
         {.note = A_5, .timeMs = 188},
         {.note = F_5, .timeMs = 188},
         {.note = A_5, .timeMs = 188},
+        {.note = SILENCE, .timeMs = 188},
+        {.note = SILENCE, .timeMs = 188},
     },
-    .numNotes = 266,
-    .shouldLoop = false
+    .numNotes = 268,
+    .shouldLoop = true
```

Comment:
He said the song is never actually used in tiltrads, so it wouldn't affect the mode in release firmware. I'd still like to know if it'd be cool with him for musical purposes / use in 1.1 tiltrads

Reply:
From @jt-moriarty:
> I personally don't have a problem with changing gameMusic since I don't use it, but I'm curious about the added silence at the end while keeping it looping, my music guy made it to loop by default.

my response:
> either [@Bedrock](https://magfest.slack.com/team/U03T1AZPRQ9) or [@FuryfulFawful](https://magfest.slack.com/team/UQ9ER2GCB) discovered that looping behaved weirdly if there was no silence at the end of a song, and this was recent, like the day of the freeze or a day or two before